modular_skyrat/modules/company_imports/code/armament_datums/nakamura_tools.dm

```diff
@@ -0,0 +1,122 @@
+	lower_cost = CARGO_CRATE_VALUE * 0.25
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/multitool
+	item_type = /obj/item/multitool
+	lower_cost = CARGO_CRATE_VALUE * 0.25
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/inducer
+	item_type = /obj/item/inducer
+	lower_cost = CARGO_CRATE_VALUE * 0.25
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/magboots
+	item_type = /obj/item/clothing/shoes/magboots
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/insuls
+	item_type = /obj/item/clothing/gloves/color/yellow
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+// Advanced stuff like power tools and holofans
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced
+	subcategory = "Advanced Engineering Equipment"
+	interest_required = PASSED_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/skillchip
+	item_type = /obj/item/skillchip/job/engineer
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/ranged_analyzer
+	item_type = /obj/item/analyzer/ranged
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/forcefield
+	item_type = /obj/item/forcefield_projector
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/atmos_fan
+	item_type = /obj/item/holosign_creator/atmos
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/powerdrill
+	item_type = /obj/item/screwdriver/power
+	lower_cost = CARGO_CRATE_VALUE
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/jaws
+	item_type = /obj/item/crowbar/power
+	lower_cost = CARGO_CRATE_VALUE
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/hugewelder
+	item_type = /obj/item/weldingtool/hugetank
+	lower_cost = CARGO_CRATE_VALUE
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+// Overpriced experimental or gimmick tools
+
+/datum/armament_entry/company_import/nakamura_tooling/experimental
+	subcategory = "Experimental Engineering Equipment"
+	interest_required = HIGH_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_BIG
+
+/datum/armament_entry/company_import/nakamura_tooling/experimental/sprayon_insuls
+	item_type = /obj/item/toy/sprayoncan
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 4
+
+/datum/armament_entry/company_import/nakamura_tooling/experimental/red_screwdriver
```

Comment:
maybe throw in the 'advanced tools' from that one space ruin that barely spawns? the bootleg alien tools

Reply:
Not sold on having the caravan tools, definitely don't want the advanced ones